src/core/linux/SDL_udev.c

```diff
@@ -38,6 +38,17 @@
 
 static const char *SDL_UDEV_LIBS[] = { "libudev.so.1", "libudev.so.0" };
 
+#ifdef SDL_UDEV_DYNAMIC
+
+SDL_ELF_NOTE_DLOPEN(
+    "events-udev",
+    "Support for events through libudev",
+    SDL_ELF_NOTE_DLOPEN_PRIORITY_SUGGESTED,
```

Comment:
I'd strengthen this to `RECOMMENDED`.

Reply:
I also added the fallback libs to the note.

Reply:
I think you were correct to list all the fallbacks in a single note here (the `OR` pattern).